### PR TITLE
remove min replicas to ibievents and isobuild

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

remove readiness for isobuilds

Fixes # ( THEEDGE-2216  THEEDGE-2217)

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Comment by ldjebran** [comment]
remove do not merge as automation is ok now

**Review by de1987 — APPROVED** [review]
LGTM

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -177,7 +177,7 @@ objects:
             cpu: 125m
             memory: 128Mi
     - name: ibvents
-      minReplicas: 1
+      minReplicas: 0
```
Comment (loadtheaccumulator) [inline deploy/clowdapp.yaml:180]: This has to be 1. ibvents is production and responsible for resuming interrupted builds and marking stale builds as ERROR.

**Review by loadtheaccumulator — CHANGES_REQUESTED** [review]

**Review by loadtheaccumulator — APPROVED** [review]

**Comment by omaciel** [comment]
This PR is still labeled as DRAFT. BTW the description says ibevents and isobuilds will be updated but only one has been updated

**Pull request merged**